spec/compiler/DocumentParserSpec.js

```diff
@@ -140,6 +155,13 @@ describe("DocumentParser", function() {
     });
   });
 
+  describe("void tags", function() {
+    it("base", function() {
```

Comment:
nit: try to make the `it` description describe whats tested, e.g.: `it("ends void tags  automatically")`